Author: Daniel Baumann <daniel.baumann@progress-technologies.net>
Description:
 Setting default suite to squeeze in debian template (Closes: #600456).

diff -Naurp lxc.orig/templates/lxc-debian.in lxc/templates/lxc-debian.in
--- lxc.orig/templates/lxc-debian.in	2010-11-06 21:40:36.574920897 +0100
+++ lxc/templates/lxc-debian.in	2010-11-06 23:20:11.051807258 +0100
@@ -20,7 +20,7 @@
 # License along with this library; if not, write to the Free Software
 # Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
 
-SUITE=${SUITE:-lenny}
+SUITE=${SUITE:-squeeze}
 
 configure_debian()
 {
